package/src/components/Message/MessageSimple/ReactionList.tsx

```diff
@@ -172,6 +183,7 @@ const ReactionListWithContext = <
   } = useTheme();
 
   const opacity = useSharedValue(0);
+  const { targetedMessage } = useTargetedMessage(message.id);
```

Comment:
Is this working? Because we set the targetedMessage in Channel component. If you use the hook explicitely here, `targetedMessage` will always be the current message.

Reply:
Yes, this works fine as I am checking the condition later if targetedMessage === message.id later.

Reply:
But if you check the hook, the initial value of `targetedMessage` will be `message.id`. And you never update that value in this hook using `setTargetedMessage`.

Reply:
So `targetedMessage === message.id` will be true always on load, and will be reset by setTimeout within the hook.

Reply:
Ideally, can we pass the prop through Channel and pass it under MessageContext and then use it here? Will that work?